### PR TITLE
Gnome Circle Tweaks

### LINKED ISSUE #268: GNOME Circle Tweaks
Source: https://gitlab.gnome.org/Teams/Circle/-/issues/128#note_1755755

- [x] Filename -> File Name
- [x]  Is there a way to remove downloads? I couldn't find one, at least. Would be nice to have some kind of menu on the rows for that, or selection mode.
    - Awaiting clarification
- [x] This menu looks like a save button, but it's something totally different. Given that there's not much in there I think these actions could all just go into the primary menu
    - Awaiting clarification  
    - image
- [x] I'd use a different icon for the logs thing here, e.g. maybe utilities-terminal?
    -  image
- [x] The welcome screen looks pretty good now, but the icon needs the icon-dropshadow class, and the labels could be more inviting. How about "Download Videos -- Add a video URL to start downloading"
- [x] Feature idea: If there's a recognized URL in the clipboard, show a toast allowing people to add it directly, like Fragments does

### PR DESCRIPTION
**Pull request by nlogozzo**
Closes #268

### CODE REVIEW

File: NickvisionTubeConverter.Shared/Resources/Strings.resx [hunk 1]
```diff
@@ -225,7 +225,7 @@
 	</data>
 
 	<data name="NoDownloads.Description" xml:space="preserve">
-		<value>Add a download to get started.</value>
+		<value>Add a video URL to start downloading.</value>
```
Comment (fsobolev) [inline NickvisionTubeConverter.Shared/Resources/Strings.resx:228]: How about "Add a video, audio or playlist URL to start downloading."?

File: NickvisionTubeConverter.Shared/Resources/Strings.resx [hunk 2]
```diff
@@ -422,7 +422,7 @@
 	</data>
 
 	<data name="NoDownloadsRunning" xml:space="preserve">
-		<value>No downloads running</value>
+		<value>Download Videos</value>
```
Comment (fsobolev) [inline NickvisionTubeConverter.Shared/Resources/Strings.resx:425]: "Download media"?

**Review by fsobolev — CHANGES_REQUESTED** [review]

File: NickvisionTubeConverter.Shared/Resources/Strings.resx (hunk 1 above)
Comment (nlogozzo) [reply NickvisionTubeConverter.Shared/Resources/Strings.resx:228]: That works :)

File: NickvisionTubeConverter.Shared/Resources/Strings.resx (hunk 2 above)
Comment (nlogozzo) [reply NickvisionTubeConverter.Shared/Resources/Strings.resx:425]: Sounds better!

**Pull request closed without merging**